ch_util/rfi.py

```
@@ -458,7 +476,18 @@ def frequency_mask(
     # Broadcast to get the output mask
     mask = np.zeros(np.broadcast(freq_centre, timestamp).shape, dtype=bool)
 
-    for (start_time, end_time), (fs, fe) in BAD_FREQUENCIES:
+    # Time-dependent static RFI flagging
+    if instrument is None:
+        instrument = "chime"
+
+    if timestamp is None:
+        timestamp = time.time()
+    bad_freq = bad_frequencies[instrument]
+
+    if bad_freq is None:
```

This should probably just no-op.

---

I don't understand... just fail silently?

---

Yeah, this is up for debate. It depends on what you think the behaviour is for when you give it an instrument name it recognises but no one has put in the RFI mask for yet. Fail loudly, or no-op? I think I could go either way, but I figured the most useful would be the latter.